packages/Breadcrumbs/Item/Item.modules.scss

```diff
@@ -36,14 +36,3 @@
     margin: 0;
   }
 }
-
```

Comment:
On line 3, you can change the import statement to use tilde imports now that `shared-typography` is a versioned package.